### DISCUSSION TITLE
[CH5] 표현식

### DISCUSSION THREAD

**Discussion by innerstella** [innerstella/ES2024#9] [Q&A] [answered]
아래 코드문에서 표현식의 수는 몇개인가요? 

```
var x;
x = 1 + 2;
```

**Comment by hsmurf** [comment, marked as answer]
var x 는 변수 선언문 이므로 표현식이 아님

첫 번째 표현식: 1 + 2
이 표현식은 두 개의 숫자를 더한 값을 반환. 즉, 1 + 2는 3으로 평가.

두 번째 표현식: x = 1 + 2
이 표현식은 변수 x에 1 + 2의 결과를 할당하는 할당 연산. 1 + 2는 위에서 언급한 대로 3으로 평가되므로, 이 표현식은 x 변수에 3을 할당.

따라서 이 코드는 두 개의 표현식으로 구성
